src/main/java/uk/gov/hmcts/sptribs/common/event/CreateCase.java

```diff
@@ -142,6 +156,26 @@ public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details,
             .build();
     }
 
+    private void setDefaultCaseDetails(CaseData data) {
```

Comment:
Is it worth importing this from CicCreateCaseEvent? Or vice versa?

Reply:
I think it's better to keep them separately as one is used for our frontend and another for ExUI. So, it will be easy to separate them if we need in the future.